src/user/UserCredentials.tsx

```diff
@@ -92,6 +108,101 @@ const DisplayDialog: FunctionComponent<DisplayDialogProps> = ({
   );
 };
 
+const CredentialsResetActionMultiSelect = (props: {
+  form: UseFormMethods<CredentialResetForm>;
+}) => {
+  const { t } = useTranslation("users");
+  const [open, setOpen] = useState(false);
+  const { form } = props;
+  const { control } = form;
+
+  return (
+    <FormGroup
+      label={t("resetActions")}
+      labelIcon={
+        <HelpItem
+          helpText="clients-help:resetActions"
+          fieldLabelId="resetActions"
+        />
+      }
+      fieldId="actions"
+    >
+      <Controller
+        name="actions"
+        defaultValue={[]}
+        control={control}
+        render={({ onChange, value }) => (
+          <Select
+            toggleId="actions"
+            variant={SelectVariant.typeaheadMulti}
+            chipGroupProps={{
+              numChips: 3,
+            }}
+            onToggle={(open) => setOpen(open)}
+            isOpen={open}
+            selections={value.map((o: string) => o)}
+            onSelect={(_, selectedValue) =>
+              onChange(
+                value.find((o: string) => o === selectedValue)
+                  ? value.filter((item: string) => item !== selectedValue)
+                  : [...value, selectedValue]
+              )
+            }
+            onClear={(event) => {
+              event.stopPropagation();
+              onChange([]);
+            }}
+            aria-label={t("resetActions")}
+          >
+            {Object.values(RequiredActionAlias).map((action, index) => (
+              <SelectOption
+                key={index}
+                value={action}
+                data-testid={`${action}-option`}
+              >
+                {t(action)}
+              </SelectOption>
+            ))}
+          </Select>
+        )}
+      />
+    </FormGroup>
+  );
+};
+
+const LifespanField = (props: {
+  form: UseFormMethods<CredentialResetForm>;
+}) => {
+  const { t } = useTranslation("users");
+
+  const { form } = props;
+  const { control } = form;
+
+  return (
```

Comment:
You could also put it all in one line like this:
```suggestion
const LifespanField = ({
  form: { control },
}: {
  form: UseFormMethods<CredentialResetForm>;
}) => {
  const { t } = useTranslation("users");

  return (
```